hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -70,6 +75,31 @@
 	private Id proxy;
 	private int autoAssociationMetadata;
 
+	//TODO: Add JavaDoc if approved
+	public void updateFromGrpcContract(
+			final Optional<JKey> newAdminKey,
+			final Optional<AccountID> newProxy,
+			final Optional<Duration> newAutoRenewPeriod,
+			final Optional<Timestamp> newExpirationTime,
+			final Optional<String> newMemo,
+			final boolean affectsExpiryOnly) {
```

Comment:
Instead of passing `affectsExpiryOnly` (which is calculated based on the arguments in this method) why don't we do:
```Java
affectsExpiryOnly = newAdminKey.isEmpty() && newProxy.isEmpty() ... 
``` 

That way, we won't be passing a boolean property to the model that could be computed based on the arguments provided.

Reply:
With one quick note: 
```
	boolean affectsExpiryOnly(ContractUpdateTransactionBody op) {
		return !(op.hasProxyAccountID()
				|| op.hasFileID() // <- note here
				|| affectsMemo(op)
				|| op.hasAutoRenewPeriod()
				|| op.hasAdminKey());
	}
```

Reply:
Yes! So file FILE ID is completely ignored! It must be deprecated